keyboards/deltasplit75/protosplit/protosplit.c

```diff
@@ -0,0 +1,32 @@
+#include "deltasplit75.h"
+
+#ifdef AUDIO_ENABLE
+    float tone_startup[][2] = SONG(STARTUP_SOUND);
+    float tone_goodbye[][2] = SONG(GOODBYE_SOUND);
+#endif
+
+void matrix_init_kb(void) {
+
+    #ifdef AUDIO_ENABLE
+        _delay_ms(20); // gets rid of tick
+        PLAY_NOTE_ARRAY(tone_startup, false, 0);

```

Comment:
The startup song is actually played by default - you don't need to have any of this in here :)